src/Bridge/Symfony/Bundle/Resources/config/api.xml

```diff
@@ -19,6 +19,15 @@
             <argument type="service" id="api_platform.metadata.resource.metadata_factory" />
         </service>
 
+        <service id="api_platform.route_name_resolver" class="ApiPlatform\Core\Bridge\Symfony\Routing\RouteNameResolver" public="false">
+            <argument type="service" id="api_platform.router" />
+        </service>
+
+        <service id="api_platform.route_name_resolver.cached" class="ApiPlatform\Core\Bridge\Symfony\Routing\CachedRouteNameResolver" decorates="api_platform.route_name_resolver" decoration-priority="-10" public="false">
```

Comment:
To convey the intention that this should run dead last, and so that any user-declared decorators would be correctly wrapped by default (decoration priority 0). It's good to have, and consistent with the decorated metadata factory services...